src/components/landingpage/UserChoicePanelRight.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 
 import { ILanguage } from '@src/interfaces';
-import { LanguagesList } from '@components/languagechoices/LanguagesList';
+import { LanguagesList } from '@pages/LanguagesList';
```

Comment:
I see a new path alias being used here but the declaration for the same hasn't been done anywhere. Refer to `tsconfig.paths.json`

Reply:
Done